Writing_PRs.md

```diff
@@ -41,6 +41,21 @@ background that led to this change, so be sure to provide it for them.
 Here's an example that hits many of these points and is nicely formatted for reading:
 https://github.com/dimagi/commcare-hq/pull/20874
 
+# PR Labels
+
+PRs are labeled to help the product team sift through changes and decide what to communicate to different user audiences. Every PR must have one of the following product-related labels:
+* `product/invisible` Changes with no user-facing effects.
```

Comment:
My understanding is that "invisible" also includes bug fixes, some of which have user-facing effects, so this description seems incomplete. Would it make sense to include a note about how it will (not) be used? Something like
```suggestion
* `product/invisible` Bug fixes and changes with no user-facing effects. Will not be announced in the weekly change log.
```
This was recently a point of confusion when I was trying to explain it to a new member of the team.

Reply:
I'll check in with Andrea.